### PR TITLE
Some Mode field can be strings

### LINKED ISSUE #27: if Video, Audio Track->format is string, value is invalid.
my video file's video, audio track's format is string.

/src/Checker/ModeChecker.php 

```
    public function create($rateMode)
    {
        if (is_string($rateMode)) $rateMode = array($rateMode); // <- add this line
        if (!isset($rateMode[1])) {
            $rateMode[1] = $rateMode[0];
        }
        $mode = new Mode($rateMode[0], $rateMode[1]);

        return $mode;
    }
```

### PR DESCRIPTION
**Pull request by mhor**
fix #27

### CODE REVIEW

**Comment by mhor** [comment]
@yupmin could you check if it's look good to you?

File: src/Checker/ModeChecker.php [hunk 1]
```diff
@@ -13,12 +13,15 @@ class ModeChecker extends AbstractAttributeChecker
...
+            $rateMode = array($rateMode);
+        }
+
         if (!isset($rateMode[1])) {
             $rateMode[1] = $rateMode[0];
         }
-        $mode = new Mode($rateMode[0], $rateMode[1]);
 
-        return $mode;
+        return  new Mode($rateMode[0], $rateMode[1]);
```
Comment (GrahamCampbell) [inline src/Checker/ModeChecker.php]: cs

Comment (mhor) [inline src/Checker/ModeChecker.php]: @GrahamCampbell what is the issue?

Comment (GrahamCampbell) [inline src/Checker/ModeChecker.php]: there should be exactly one space following `return`, not two

Comment (mhor) [inline src/Checker/ModeChecker.php]: Oh nice catch!

File: src/Checker/ModeChecker.php [hunk 2]
```diff
@@ -13,12 +13,15 @@ class ModeChecker extends AbstractAttributeChecker
      */
     public function create($rateMode)
     {
+        if (!is_array($rateMode)) {
```
Comment (GrahamCampbell) [inline src/Checker/ModeChecker.php]: You need to update the phpdoc then

Comment (GrahamCampbell) [inline src/Checker/ModeChecker.php]: Why not just to `$rateMode = (array) $rateMode`, then you don't need any checks.

**Comment by mhor** [comment]
@GrahamCampbell thanks for review

File: src/Checker/ModeChecker.php [hunk 3]
```diff
@@ -7,18 +7,18 @@
...
     /**
-     * @param array $rateMode
+     * @param array|string $rateMode
      *
      * @return Mode
      */
     public function create($rateMode)
     {
+        $rateMode = (array) $rateMode;
         if (!isset($rateMode[1])) {
```
Comment (GrahamCampbell) [inline src/Checker/ModeChecker.php]: would be cleaner to leave a newline before this if statement

**Comment by yupmin** [comment]
@mhor Yes. I Tested this. It works well.

**Comment by mhor** [comment]
@yupmin tag as 1.43

**Pull request merged**